test/unit/app/services/bulk_republisher_test.rb

```diff
@@ -201,4 +201,67 @@ class BulkRepublisherTest < ActiveSupport::TestCase
       BulkRepublisher.new.republish_all_published_organisation_about_us_pages
     end
   end
+
+  describe "#republish_all_by_type" do
+    context "for editionable document types" do
+      document_types = %w[CaseStudy
```

Comment:
We probably don't need to loop over all these if we're stubbing the `non_editionable_document_types` return value. Let's just use one of these. Should we stub `non_editionable_document_types` further up?